### PR TITLE
Review code integrate linters htmls css best practice

### PR DESCRIPTION
**Pull request by evillaz**
This branch's purpose is to do the reviewing in trials project to check if were applied:
best practices in HTML code.
Semantic HTML tags.
CSS selectors correctly.
CSS box model.
Perform a code review for yourself on a past project.

### CODE REVIEW

**Comment by evillaz** [comment]
[⭕️]Used semantic HTML tags.

[⭕️]Used CSS selectors correctly.

[⭕️]Used CSS box model.


 [❌ ]  Not correct flow used
 [❌ ] Linters errors
 [❌ ]  Repo not documented in a professional way

**Comment by evillaz** [comment]
- Code didn't get to pass the linters

    - Indentation and spacing errors.
 **Kindly take the time to correct the linters flags on the styling.**

**Comment by evillaz** [comment]
- Workflow was not followed
    - All the updates and corrections were directly changed to the main branch, not following the correct workflow.
**I think It would be better if you create a new branch for any alterations or modifications before merging to the main branch. Make sure that you are always working in a feature branch!**

**Comment by evillaz** [comment]
Repo was not documented in a professional way
**Therefore make sure you always use descriptive names for your branches and make sure that your 'feature branch' is up-to-date with the 'main branch' in your repository.
Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request.
Use descriptive commit messages**

File: fcc-test-suite-template-master/README.md [hunk 1]
```diff
@@ -1,5 +1,234 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Personal Portfolio] <a name="about-project"></a>
+
+**[Personal portfolio]** is a html project with css styling used to introduce ouselfs as programmer through a Portfolio
```
Comment (Olamarx) [inline fcc-test-suite-template-master/README.md:37]: - [ ] This is an inline comment. I would advise you to do something similar.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @evillaz, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Grammatically correct sentences ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] I would advise you to make at least one inline comment just as I have done. You can find how to do it [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)
- [ ] I would advise you to give reasons for your required changes. I can see you commented on linters and other things but you did not give a reason for the student to fix the linters. This is for the student to understand your point of view as the reviewer and not be acting blindly.
- [ ] Kindly make your tone friendly and encouraging not condemning. In the aspect you wrote `workflow was not followed`, I would advise you to make it a more friendly and correctable tone. something like: 'What do you think about following the workflow?', 'I would advise you to follow the workflow' etc.
image
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by evillaz** [comment]
- It seems like linters flagged out some errors!

**Linters check on the basic syntax as well as apply a set of rules to the code that look for problematic patterns or signs of inefficiency. [These] errors are usually for:**
- Correct use of indentation
- Correct spacing 
- Correct or expecting units 
- Correct use of CSS selectors
 **Kindly take the time to correct the linters flags on the styling. As shown on the details tab **
 
linters

**Comment by evillaz** [comment]
What do you think about checking on the correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)?

I would strongly suggest that you could give it a read to ensure that:
- Before you start working on a new feature, you should create a new feature branch that is based on the main branch. You will use this branch for all your work.(*Remember that your branch name should be descriptive*)
- You will need to create a pull request to compare your feature branch with the main branch and submit the link to that pull request via your Dashboard.
- **[IMPORTANT!!] You should not merge your pull requests to the main branch of your project without a code review that approves your changes.**

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @evillaz, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
- Proper Usage of the Github review feature ✔️ 
- Use Of emoji ✔️  


### Changes Required
- As suggested by the previous reviewer I would suggest you kindly make use of an inline comment as currently as I can't see you having any inline comment. Kindly check this [link](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) for more details.

> N.B: You can always use the example given to you by the previous reviewer in his comment

- As suggested by the previous reviewer I would suggest you make your comment more polite and be in form of a suggestion rather than given an order. 
For example:
```
  - This is a polite suggestion: "I think it would be better to use semantic tags"
  - This reads as an order: "Use semantic tags".
```
- As Suggested by the previous reviewer I would suggest you give a reason `Why` you suggest a change: `

 For example: (A good review)
```
* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*
```

- I would kindly suggest you make use of a friendly gif in your review as this gives your review a kind of friendly attraction to the author of the project you are reviewing.

- I would suggest you make use of a general review the way reviewers use to leave it in your project. like the one that I used.

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: fcc-test-suite-template-master/README.md [hunk 2]
```diff
@@ -1,5 +1,234 @@
...
+  - First i tried to address each one of the requirements and solve them individually to keep integrating them as i was progressing.
+  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (evillaz) [inline fcc-test-suite-template-master/README.md:232]: What do you think it's missing in applying and stating a License being used?
Exactly! If you are stating a license is being used in your project make sure you added it as well into your repo's directory
I would kindly suggest that you could check on how to [add a license](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to your repository

**Comment by evillaz** [comment]
# Status: Required Changes :recycle:
Hi @evillaz 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Semantic HTML tags used
- ✔️ CSS selectors used correctly.
- ✔️ CSS box model applied
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._
It seems like linters flagged out some errors!

Why do linters flags usually flag on your repository? Linters check on the basic syntax as well as apply a set of rules to the code that look for problematic patterns or signs of inefficiency. [These] errors are usually for:

- Wrong use of indentation

- Missing or over spacing

- Correct or unit expectation

- Correct use of CSS selectors

**Why don't you take a while and check on the linters flags on the styling? As shown on the details tab. That way would improve your CSS file structure as you apply the best practices on HTML & CSS **
linters
What do you think about checking on the correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)? That way you could ensure that all the changes on your project won't be updated to the main branch until you get the clearance from the reviewers**

I kindly suggest that you could give it a read to ensure that:

Before you start working on a new feature, you should create a new feature branch that is based on the main branch. You will use this branch for all your work. (Remember that your branch name should be descriptive)
You will need to create a pull request to compare your feature branch with the main branch and submit the link to that pull request via your Dashboard.
[IMPORTANT!!] You should not merge your pull requests to the main branch of your project without a code review that approves your changes.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: fcc-test-suite-template-master/README.md (hunk 2 above)
Comment (evillaz) [inline fcc-test-suite-template-master/README.md:232]: What do you think it's missing in applying and stating a License being used?
Exactly! If you are stating a license is being used in your project make sure you added it as well into your repo's directory
I would kindly suggest that you could check on how to [add a license](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to your repository

**Review by evillaz — COMMENTED** [review]
s

File: fcc-test-suite-template-master/README.md (hunk 2 above)
Comment (evillaz) [inline fcc-test-suite-template-master/README.md:232]: What do you think it's missing in applying and stating a License being used?
Exactly! If you are stating a license is being used in your project make sure you added it as well into your repo's directory
I would kindly suggest that you could check on how to [add a license](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to your repository

**Review by evillaz — COMMENTED** [review]
# Status: Required Changes :recycle:
Hi @evillaz 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Semantic HTML tags used
- ✔️ CSS selectors used correctly.
- ✔️ CSS box model applied
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._
It seems like linters flagged out some errors!

Why do linters flags usually flag on your repository? Linters check on the basic syntax as well as apply a set of rules to the code that look for problematic patterns or signs of inefficiency. [These] errors are usually for:

- Wrong use of indentation

- Missing or over spacing

- Correct or unit expectation

- Correct use of CSS selectors

**Why don't you take a while and check on the linters flags on the styling? As shown on the details tab. That way would improve your CSS file structure as you apply the best practices on HTML & CSS **
linters
What do you think about checking on the correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)? That way you could ensure that all the changes on your project won't be updated to the main branch until you get the clearance from the reviewers**

I kindly suggest that you could give it a read to ensure that:

Before you start working on a new feature, you should create a new feature branch that is based on the main branch. You will use this branch for all your work. (Remember that your branch name should be descriptive)
You will need to create a pull request to compare your feature branch with the main branch and submit the link to that pull request via your Dashboard.
[IMPORTANT!!] You should not merge your pull requests to the main branch of your project without a code review that approves your changes.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by evillaz** [comment]
hi, I wasn't aware of some definitions on the 1st review can you check the
new review I posted on the pull request, please? so I would request a 3rd
review D: I don't want to be missing any of the required changes

**Comment by evillaz** [comment]
@brainconnect93 <https://github.com/brainconnect93>

**Review by Ibizugbe — APPROVED** [review]
Hi @evillaz 👋, 

BigTvGIF (2)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅
- Nice work putting the suggestions given by the previous reviewers into consideration👍
- You have made use of emojis and sticker to make your comments look more friendlu👍
- Your use images to make your comments more descriptive👍

## Optional suggestions
_no optional suggestion_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**